Lib/idlelib/configdialog.py

```diff
@@ -88,6 +88,7 @@ def __init__(self, parent, title='', _htest=False, _utest=False):
         self.attach_var_callbacks()  # Avoid callbacks during load_configs.
 
         if not _utest:
+            self.grab_set()
```

Comment:
Since it will not be called, no need.  I just read the UNM Shipman doc "Widget w grabs all events for w's application. If there was another grab in force, it goes away. See Section 54, “Events” for a discussion of grabs."  Seems to be related to dialog being modal.   Did not see anything in Sect.54 though.  I need to make sure I know about *all* the calls in various inits.